src/data_client.cpp

```diff
@@ -40,12 +40,14 @@ namespace metricq
 DataClient::DataClient(const std::string& token, bool add_uuid) : Connection(token, add_uuid)
 {
     TimePoint starting_time = Clock::now();
-    register_management_callback("discover", [token, starting_time](const json&) {
-        json response;
-        response["alive"] = true;
-        response["currentTime"] = Clock::format_iso(Clock::now());
-        response["startingTime"] = Clock::format_iso(starting_time);
-        return response;
+    register_management_callback("discover", [starting_time](const json&) -> json {
```

Comment:
I think you can initialize the `starting_time` directly in the lambda instead of capture a local variable.

```suggestion
    register_management_callback("discover", [starting_time = Clock::now()](const json&) -> json {
```

As stating_time is only in the local scope, that should be fine.

Reply:
Oh, I did not know one could do that. Sure, will do.